reference-adapter/mgramseva-ifix-adapter/src/main/java/org/egov/ifix/utils/DataWrapper.java

```diff
@@ -79,4 +85,65 @@ public String getFiscalEventDetailsInformation(List<FiscalEvent> fiscalEventList
 
         return fiscalMessage.toString();
     }
+
+    /**
+     * @return
+     */
+    public Long getValidSearchIntervalTime() {
+        String plainTime = applicationConfiguration.getIfixFiscalEventSearchIntervalTime();
+
+        if (!org.apache.commons.lang3.StringUtils.isNumeric(plainTime)) {
+            log.error(">>>>> Ifix fiscal event search interval time value is invalid");
+            throw new GenericCustomException(TIME, "Ifix fiscal event search interval time value is invalid");
+        }else {
+            return Long.parseLong(plainTime) * 1000;
+        }
+    }
+
+    /**
+     * @param plainTextTime
+     * @return
+     */
+    public Long getMilliSecondTime(String plainTextTime) {
+        long oneHourMilliSecond = 60L * 60L * 1000L;
+        long totalMilliSecond = 0L;
+
+        if (isValidSearchIntervalTime(plainTextTime)) {
+            String[] timeArray = plainTextTime.split(" ");
+
+            String paramValue = timeArray[0];
+            totalMilliSecond += Long.parseLong(paramValue) * 24 * oneHourMilliSecond;
```

Comment:
Why are we having this custom code?
Weren't you able to find any library that could do it for us?

Reply:
It is a utility, which can be used independently, whenever we want. It converts git time format to millisecond i.e. 1d 2h 30m.

I could remove but future reference I keep it. Should I remove?

Reply:
Is this custom logic required or not? If you are converting 2h 30m to milliseconds by some library then remove this code. 
Any unnecessary extra code is more confusing rather than helpful in the future.

Reply:
done